pkg/install/install.go

```diff
@@ -63,6 +63,10 @@ func (obj *installer) install(
 		pkg.SetLabels(map[string]string{
 			"packages.glasskube.dev/auto-update": "true",
 		})
+	} else {
```

Comment:
Is this change really necessary? I think everything should work without it too, right? Or was there some problem related to that?

Reply:
I thought of removing it later but I forgot it was accidental